prisma/migrations/20241020070237_add_invite_link_to_team.sql/migration.sql

```diff
@@ -0,0 +1,12 @@
+-- AlterTable
+DO $$ 
+BEGIN
+    IF EXISTS (SELECT 1 FROM information_schema.columns WHERE table_name = 'Team' AND column_name = 'inviteLink') THEN
+        ALTER TABLE "Team" DROP COLUMN "inviteLink";
+    END IF;
+END $$;
+
```

Comment:
I understand that you changed the schema locally for yourself, but please remove this from the migration

Reply:
you mean in my PR I should not be add this file?